packages/pluggableWidgets/slider-web/CHANGELOG.md

```diff
@@ -4,3 +4,7 @@ All notable changes to this widget will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Added
+
+- We introduced new property: "Tooltip always visible". If this property is enabled, tooltip is always visible on the screen.
```

Comment:
Not necessary as this widget is unreleased yet, so you can remove this CHANGELOG change.

For next time: We don't have to be so detailed here, something like "We introduced a new widget property to always show the tooltip on the screen." is fine. Any more details should be covered in the docs.

Reply:
@Keraito, removed